CleanWeather/Scenes/CityForecast/CityForecastTableViewCell/CellManager.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  CellManager.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 07/01/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+enum CellManagerItemType {
+
+    case hourlyForecast
+    case dailyForecast
+}
+
+protocol CellManagerItem {
+
+    var type: CellManagerItemType { get }
+    var rowCount: Int { get }
+}
+
+final class CellManager: NSObject {
+
+    private var items = [CellManagerItem]()
+}
+
+extension CellManager: UITableViewDataSource {
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return items.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items[section].rowCount
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let item = items[indexPath.section]
+        switch item.type {
```

Comment:
In most of cases if you have switch case you should put each cases into separate methods